src/plugins/discover/public/application/components/chart/download_csv_button/index.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
why we add a Elasticsearch license here?

Reply:
ah bad copy and pasta job haha, will remove